analyzer/src/funTest/kotlin/managers/SpdxDocumentFileFunTest.kt

```diff
@@ -52,7 +55,17 @@ class SpdxDocumentFileFunTest : StringSpec() {
             actualResult shouldBe expectedResult
         }
 
-        // TODO: Add a test for "package.spdx.yml".
+        "mapDefinitionFiles() removes SPDX documents that do not describe a project" {
+            val projectFile = projectDir.resolve("project/project.spdx.yml")
+            val packageFile = projectDir.resolve("package/libs/curl/package.spdx.yml")
+
+            val definitionFiles = listOf(projectFile, packageFile)
+
+            val result = createSpdxDocumentFile().mapDefinitionFiles(definitionFiles)
+
+            result shouldHaveSize 1
```

Comment:
Now seeing this again, what do you think about dropping this assertion as it's clearly implied by the following assertion?